### DISCUSSION TITLE
Does autoupdate in tauri will update my application in background?

### DISCUSSION THREAD

**Discussion by Pankaj-SinghR** [tauri-apps/tauri#7536] [Q&A] [answered]
Hi Team, i just want to know does autoupdate in tauri update my application in background or it will show a notification to update when user will open the application ?

**Comment by FabianLars** [comment, marked as answer]
Depends on the settings, if you keep the default dialog option enabled then it will show a dialog on app start asking to update. If you disable it you have to implement the logic yourself which means you can download and install whenever you want.
